dev/source/docs/style-guide.rst

```diff
@@ -521,6 +521,27 @@ Bit fields are generally frowned upon
 
 Using bit fields reduces RAM usage but can considerably increase flash usage, as to extract a boolean truth value from a bit field requires more machine instructions.  If the variable is frequently accessed then this can be a LOT of flash.
 
+**Not preferred:**
+
+::
+
+   class Foo() {
+   private:
+       uint8_t should_fly  : 1;
+       uint8_t should_grow : 1;
```

Comment:
humm, I not sure that is clearer. In the text you are speaking of bool and here you are using uint8_t . Isn't using bitfields on uint8_t for packing a structure a right thing ?

Reply:
Changed, thanks.